docs/DeveloperGuide.md

```diff
@@ -293,7 +288,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a …​                | I want to …​                                      | So that I can…​                                                                                            |
 |----------|------------------------|---------------------------------------------------|------------------------------------------------------------------------------------------------------------|
-| `* * *`  | first time shop owner  | can get help on using the application             | I can quickly get up to speed with all the commands                                                        |
+| `* * *`  | first time shop owner  | learn how to use the application                  | I can familiarize myself with all the commands                                                             |
```

Comment:
i feel like both "learn how to use the application" and "familiarise myself with all the commands" could fall under the "I want to..."

i think the "so that i can" can be "I can effectively use the commands to store and manage my suppliers and deliveries"